pcdsdevices/epics_motor.py

```diff
@@ -625,6 +625,35 @@ class OffsetMotor(OffsetMotorBase):
     motor = FCpt(IMS, '{self._motor_prefix}')
 
 
+class OffsetIMSWithPreset(OffsetMotorBase):
+    """
+    Offset IMS with an additiona Offset _SET PV.
+
+    This motor puts to an additional PV (_SET) during `set_current_postion`.
+    """
+    motor = FCpt(IMS, '{self._motor_prefix}')
+    offset_set_pv = FCpt(EpicsSignal, '{self._prefix}:_SET', kind='normal')
```

Comment:
```suggestion
    offset_set_pv = FCpt(EpicsSignal, '{self._prefix}_SET', kind='normal')
```

Reply:
ohh thank you @ZLLentz - I think my brain is just so tired of this LODCM object that does not want to function well hahah

Reply:
Honestly, same, I want to merge and forget